OpenRA.Mods.Common/Scripting/Media.cs

```diff
@@ -20,6 +20,9 @@ public static class Media
 	{
 		public static void PlayFMVFullscreen(World w, string movie, Action onComplete)
 		{
+			if (w.Type == WorldType.Editor || w.IsReplay)
```

Comment:
I think you "solved" the problem with https://github.com/OpenRA/OpenRA/pull/6442 already so I can safely remove the replay check. It must have been written at a time where I still thought this project would be merged quickly and improved with subsequent pull requests instead of this endless review.